cf-usb/tasks/csm-build.sh

```diff
@@ -14,7 +14,7 @@ cd "${SIDECAR_ROOT}"
 if ! test -d vendor ; then
```

Comment:
Would this ever be true again? I believe it was put in as a transition for the move into `vendor/`.

Reply:
Well, it could happen for a test pipeline pointed to an old commit.

Reply:
But I am fine with removing this entirely.

Reply:
Done with commit 345074c906484e2478bf6d1289bd535795c298ff, in testing.